.changeset/dull-bees-obey.md

```diff
@@ -0,0 +1,28 @@
+---
+"@commercetools-backend/express": patch
+"@commercetools-backend/loggers": patch
+"@commercetools-frontend/actions-global": patch
+"@commercetools-frontend/application-components": patch
+"@commercetools-frontend/application-config": patch
+"@commercetools-frontend/application-shell-connectors": patch
+"@commercetools-frontend/application-shell": patch
+"@commercetools-frontend/assets": patch
+"@commercetools-frontend/babel-preset-mc-app": patch
+"@commercetools-frontend/browser-history": patch
+"@commercetools-frontend/constants": patch
+"@commercetools-frontend/create-mc-app": patch
+"@commercetools-frontend/cypress": patch
+"@commercetools-frontend/jest-preset-mc-app": patch
+"@commercetools-frontend/jest-stylelint-runner": patch
+"@commercetools-frontend/l10n": patch
+"@commercetools-frontend/mc-dev-authentication": patch
+"@commercetools-frontend/mc-html-template": patch
+"@commercetools-frontend/mc-scripts": patch
+"@commercetools-frontend/notifications": patch
+"@commercetools-frontend/permissions": patch
+"@commercetools-frontend/react-notifications": patch
+"@commercetools-frontend/sdk": patch
+"@commercetools-frontend/sentry": patch
+---
+
+refactor: use `combine-adapter` to be able to use `http-adapter` and `launchdarkly-adapter`
```

Comment:
Nit: I think we can keep it a bit more generic, as external users are not very familiar with this and it's anyway for internal usage only.

```suggestion
Add internal opt-in support for combining feature flag adapters.
```